stock-tracker/src/features/keyStats/component/keyStats.tsx

```diff
@@ -29,42 +30,42 @@ export const KeyStats = () => {
                   Previous Close
                 </td>
                 <td className="keystats__grid__display1__value">
-                  {response.previousClose
-                    ? Math.abs(response.previousClose.toFixed(2))
+                  {response && response.previousClose
+                    ? Math.abs(response.previousClose).toFixed(2)
                     : "N/A"}
                 </td>
               </tr>
               <tr>
                 <td className="keystats__grid__display1__name">Day Range</td>
                 <td className="keystats__grid__display1__value">
-                  {response.high && response.low
-                    ? Math.abs(response.high.toFixed(2)) +
+                  {response && response.high && response.low
+                    ? Math.abs(response.high).toFixed(2) +
```

Comment:
Noticed you have this `Math.abs(number).toFixed(2)` quite a few times. It might be worth creating a utility function at the top of this file, which does this (like you've done with `numberWithCommas`, for instance)

Reply:
Good shout will add that function in my next PR